app/models/review.rb

```diff
@@ -6,4 +6,23 @@ class Review < ApplicationRecord
   validates :comfort_index, presence: true, numericality: true, format: { with: /(?<!\S)[1-5](?!\S)/, message: "must be a number between 1 and 5" }
   validates :weather_variance, presence: true, numericality: true, format: { with: /(?<!\S)[1-5](?!\S)/, message: "must be a number between 1 and 5" }
 
+  def upvotes
```

Comment:
if you define this routine in the model, you do not need to duplicate it in the correlating serializer. You should be able to define these methods once.